### PR TITLE
{data,devel}[foss/2018b] Arrow v0.12.0, Spark v2.4.0 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#7485~~ (Hadoop)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6efcf26a066e0c87dc2f9cddbbb18c1f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7196b4cd00d9b62f440cd9d74d45f5a2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3169.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f7619b2f0096ec9892dec9d6a11fbd0e for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/fa9fdee0c06f92789393b733ce6f4001 for a full test report.

File: easybuild/easyconfigs/a/Arrow/Arrow-0.12.0-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '0.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://arrow.apache.org'
+description = "Apache Arrow (incl. PyArrow Python bindings)), a cross-language development platform for in-memory data."
```
Comment (verdurin) [inline easybuild/easyconfigs/a/Arrow/Arrow-0.12.0-foss-2018b-Python-2.7.15.eb]: @boegel nitpicking but could you use triple quotes here for consistency?

Comment (boegel) [reply easybuild/easyconfigs/a/Arrow/Arrow-0.12.0-foss-2018b-Python-2.7.15.eb]: I usually use `"..."` when the description fits on a single line (the one reason to use `"""` is if it doesn't fit on a single line)

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**